.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "javascript"
```

Comment:
`javascript` isn't listed on https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#package-ecosystem

It's listed on https://dependabot.com/docs/config-file/ but that's for version 1

Reply:
gotcha, I got confused by the multiple versions